pepper_description/urdf/pepper1.0_generated_urdf/pepper.urdf

```diff
@@ -2279,20 +2279,35 @@
   </sensor>
 </gazebo> 
 -->
+  <!-- <gazebo> -->
```

Comment:
Ah non, this urdf file should be regenerated from macro, so if we remove this code from macro than it won't be here